### PR TITLE
Move new jitm/j9jit message to the end and fix explanation prefix

### PR DESCRIPTION
**Pull request by pshipton**
New NLS messages must be added to the end of the file.
See https://github.com/eclipse-openj9/openj9/blob/master/runtime/nls/jitm/j9jit.nls#L24
J9NLS_RELOCATABLE_CODE_FSD_MISMATCH was added in the middle by
https://github.com/eclipse-openj9/openj9/commit/5fce799
from https://github.com/eclipse-openj9/openj9/pull/11554

Also fix incorrect jitm/j9jit nls explanation prefix

### CODE REVIEW

**Comment by pshipton** [comment]
@mpirvu fyi

**Comment by pshipton** [comment]
@dsouzai fyi

File: runtime/nls/jitm/j9jit.nls [hunk 1]
```diff
@@ -363,24 +355,33 @@ J9NLS_JIT_OPTIONS_LARGE_PAGE_SIZE_NOT_SUPPORTED_WITH_PAGETYPE.sample_input_6=pag
...
 # END NON-TRANSLATABLE
+
+J9NLS_RELOCATABLE_CODE_FSD_MISMATCH=FSD Enabled feature mismatch. Ignoring AOT code in shared class cache.
+# START NON-TRANSLATABLE
+J9NLS_RELOCATABLE_CODE_FSD_MISMATCH.explanation=The JVM is not running in the same FSD mode as the one that generated the code in the SCC.
+J9NLS_RELOCATABLE_CODE_FSD_MISMATCH.system_action=The compiler will not generate AOT code. AOT code from the shared class cache will not be used.
+J9NLS_RELOCATABLE_CODE_FSD_MISMATCH.user_response=Destroy and recreate the AOT code in the shared class cache for the current platform using the -Xshareclasses:reset option.
+J9NLS_RELOCATABLE_CODE_FSD_MISMATCH.link=dita:///diag/appendixes/cmdline/cmdline_x.dita#cmdline_x/xshareclasses-reset
+# END NON-TRANSLATABLE
+
```
Comment (keithc-ca) [inline runtime/nls/jitm/j9jit.nls:387]: Can you remove the extra blank lines that appear at the end of some `.nls` files?

Comment (pshipton) [reply runtime/nls/jitm/j9jit.nls]: Updated

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile zlinux jdk11

**Pull request merged**